pgdb/v1/fts.go

```diff
@@ -129,22 +110,99 @@ func lemmatizeDocs(docs []*SearchContent, additionalFilters ...jargon.Filter) []
 	return rv
 }
 
+func camelSplitDoc(docValue string, doc *SearchContent) []lexeme {
+	rv := make([]lexeme, 0, 8)
+	var word []rune
```

Comment:
perf wise, wonder if this should be using a `bytes.Buffer` with Reset() rather than `nil` the slice

Reply:
@pquerna changed in 964f2e6fbd331553c6bd49d3c749e968269e88bd